src/app/core/components/account-settings/account-settings.component.html

```diff
@@ -21,7 +21,7 @@
 <div class="settings-group">
   <div class="settings-item">
     <label>Mobile Phone
-      <i class="material-icons" [ngbTooltip]="'A mobile phone number that can recieve verification codes via text message'">help</i>
+      <i class="material-icons" [ngbTooltip]="'A mobile phone number that can recieve verification codes via text message. Permanent only supports US and Canada numbers.'">help</i>
```

Comment:
Would you mind fixing the spelling of "receive" in this message while you're here?